core/src/test/resources/org/bitcoinj/core/LegacyAddressTestDataset.txt

```diff
@@ -0,0 +1,14 @@
+12prgkB2jumVVh1iSQaSLqroe81RyaqkyR
```

Comment:
Since the idea seems to be to compare in pairs, I suggest to put a pair in each line, separated by maybe one comma or space.